src/api/.rubocop.yml

```diff
@@ -93,14 +93,18 @@ Style/NumericLiteralPrefix:
 Style/RedundantSelf:
   Enabled: true
 
-# checks for space after `!`  
+# Checks for space after `!`
 Style/SpaceAfterNot:
   Enabled: true
 
 # Checks for the presence of parentheses around ternary conditions
 Style/TernaryParentheses:
   Enabled: true
 
+# Checks for unwanted parentheses in parameterless method calls
+Style/MethodCallParentheses:
```

Comment:
because it is not the only cop disorganized, and it is impossible to know where it should go without ordering the rest. :confounded: